SIL.BuildTasks.Tests/FileUpdateTests.cs

```diff
@@ -62,6 +63,120 @@ public void GetModifiedContents_RegexTextNotMatched_Throws(string origContents,
 			Assert.That(ex.Message, Is.EqualTo($"No replacements made. Regex: '{regex}'; ReplacementText: '{replacement}'"));
 		}
 
+		[TestCase("_DATE_ _VERSION_\r\nStuff", "_DATE_", "M/yyyy", "{0} 3.2.1\r\nStuff")]
```

Comment:
This would be nicer if we'd use the [ReproducibleDateTimeProvider](https://github.com/sillsdev/libpalaso/blob/master/SIL.TestUtilities/Providers/ReproducibleDateTimeProvider.cs). Then we could do:

```suggestion
		[TestCase("_DATE_ _VERSION_\r\nStuff", "_DATE_", "M/yyyy", "January/2025 3.2.1\r\nStuff")]
```

But that would introduce a new dependency...

Reply:
Worth considering.

Reply:
I went ahead and did it. It does simplify the tests. Now that it's done, you can re-review and consider whether anything bad is likely to come of the added dependency. I think it should be okay.
<!-- Reviewable comment -ORHWTYJ2gYdPwhHgeIG:b-w7uci -->
<!-- Sent from Reviewable.io -->